daemon/cluster/configs.go

```diff
@@ -28,8 +28,7 @@ func (c *Cluster) GetConfig(input string) (types.Config, error) {
 }
 
 // GetConfigs returns all configs of a managed swarm cluster.
-// TODO(austinvazquez): decouple the daemon from client usage.
-func (c *Cluster) GetConfigs(options client.ConfigListOptions) ([]types.Config, error) {
+func (c *Cluster) GetConfigs(options swarmbackend.ConfigListOptions) ([]types.Config, error) {
```

Comment:
For these; does it work if you squash the "move" and "decouple daemom backend from client" commits, or would it then hit the "I'm Git an I am confused where it moved to" problem?